modules/display.c

```diff
@@ -3775,6 +3777,74 @@ static void mdy_blanking_start_adaptive_dimming(void)
 
 // AUTOMATIC BLANKING STATE MACHINE
 
+/** Check if blanking inhibit mode denies turning display off
+ *
+ * @return true if automatic blanking should not happen, false otherwise
+ */
+static bool mdy_blanking_inhibit_off_p(void)
+{
+    bool inhibit = false;
```

Comment:
After spending hours analyzing, refactoring, splitting up and verifying truly monstrous if-statements in mce code base - that probably all started off being "simple enough to warrant just one expression" too - I'd say: That road leads to hell.